OpenRA.Mods.Common/UtilityCommands/CheckYaml.cs

```diff
@@ -52,7 +52,21 @@ public void Run(ModData modData, string[] args)
 				var maps = new List<Map>();
 				if (args.Length < 2)
 				{
-					maps.Add(null);
+					Console.WriteLine("Testing mod: {0}".F(Game.ModData.Manifest.Mod.Title));
+					CheckGameRules(Game.ModData.DefaultRules);
```

Comment:
No, that would be wrong wrong. Not all checks are run. Especially not the `ILintMapPass`.